src/m365/planner/commands/task/task-remove.ts

```diff
@@ -0,0 +1,256 @@
+        })
+        .then(_ => cb(), (err: any) => this.handleRejectedODataJsonPromise(err, logger, cb));
+    };
+    if (args.options.confirm) {
+      removeTask();
+    }
+    else {
+      Cli.prompt({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+        message: `Are you sure you want to remove the task ${args.options.id || args.options.title}?`
+      }, (result: { continue: boolean }): void => {
+        if (!result.continue) {
+          cb();
+        }
+        else {
+          removeTask();
+        }
+      });
+    }   
+  }
+
+  private getTask(options: Options): Promise<PlannerTask> {
+    const { id, title } = options;
+
+    if(id) {
+      const requestOptions: AxiosRequestConfig = {
+        url: `${this.resource}/v1.0/planner/tasks/${id}`,
+        headers: {
+          accept: 'application/json'
+        },
+        responseType: 'json'
+      };
+      
+      return request.get<PlannerTask>(requestOptions);
+    }
+
+    return this
+      .getBucketId(options)
+      .then(bucketId => {
+        const requestOptions: AxiosRequestConfig = {
+          url: `${this.resource}/v1.0/planner/buckets/${bucketId}/tasks`,
+          headers: {
+            accept: 'application/json;odata.metadata=none'
+          },
+          responseType: 'json'
+        };
+
+        return request.get<{ value: PlannerTask[] }>(requestOptions);
+      })
+      .then(tasks => {
+        const filteredTasks = tasks.value.filter(b => title!.toLocaleLowerCase() === b.title!.toLocaleLowerCase());
+
+        
+        if (!filteredTasks.length) {
+          return Promise.reject(`The specified task ${title} does not exist`);
+        }
+
+        if (filteredTasks.length > 1) {
+          return Promise.reject(`Multiple tasks with title ${title} found: ${filteredTasks.map(x => x.id)}`);
+        }
+
+        return Promise.resolve(filteredTasks[0]);
+      });
+  }
+
+  private getBucketId(options: Options): Promise<string> {
+    const { bucketId, bucketName } = options;
+
+    if (bucketId) {
+      return Promise.resolve(bucketId);
+    }
+
+    return this
+      .getPlanId(options)
+      .then(planId => {
+        const requestOptions: AxiosRequestConfig = {
+          url: `${this.resource}/v1.0/planner/plans/${planId}/buckets`,
+          headers: {
+            accept: 'application/json;odata.metadata=none'
+          },
+          responseType: 'json'
+        };
+
+        return request.get<{ value: PlannerBucket[] }>(requestOptions);
+      })
+      .then((buckets) => {
+        const filteredBuckets = buckets.value.filter(b => bucketName!.toLocaleLowerCase() === b.name!.toLocaleLowerCase());
+        
+        if (!filteredBuckets.length) {
+          return Promise.reject(`The specified bucket ${bucketName} does not exist`);
+        }
+
+        if (filteredBuckets.length > 1) {
+          return Promise.reject(`Multiple buckets with name ${bucketName} found: ${filteredBuckets.map(x => x.id)}`);
```

Comment:
And same here, let's use os.newline. Don't forget to change your spec.ts files as well.